trader/src/backend.rs

```diff
@@ -97,11 +106,16 @@ pub fn run_backend(
                         );
                     })
                 }
-                Command::Register { symbol, faction } => rt.block_on(async {
-                    response_data_lock.register_data = UnwrapReq!(
-                        register(&config, Some(RegisterRequest::new(faction, symbol))).await,
-                        latest_cmd.1
-                    );
+                Command::Register {
+                    symbol,
+                    faction,
+                    email,
+                } => rt.block_on(async {
+                    let mut request = RegisterRequest::new(faction, symbol);
+                    request.email = Some(email);
```

Comment:
Since this isn't a required field in the API (Only used if you reserved your call sign between resets), it should probably check if the string is empty and set request.email to `None` if so. (IDK how the Space Traders API handels an empty string right now and behavior could change in the future so it's best to look out for it now)

Reply:
or something like this:
```rust
if !email.is_empty() {
    request.email = Some(email)
}
```
Since it's `None` by default